ndsl/__init__.py

```diff
@@ -1 +1,62 @@
+from .buffer import Buffer
+from .comm.boundary import Boundary, SimpleBoundary
+from .comm.communicator import CubedSphereCommunicator, TileCommunicator
+from .comm.local_comm import AsyncResult, ConcurrencyError, LocalComm
```

Comment:
AsyncResult and ConcurrencyError don't need to be root level imo

Reply:
Might by needed in `typing`